public/app/core/components/Footer/Footer.tsx

```diff
@@ -38,6 +38,11 @@ export let getVersionLinks = (): FooterLink[] => {
   const stateInfo = licenseInfo.stateInfo ? ` (${licenseInfo.stateInfo})` : '';
 
   links.push({ text: `${buildInfo.edition}${stateInfo}`, url: licenseInfo.licenseUrl });
+
+  if (buildInfo.version === '') {
+    return links;
+  }
```

Comment:
Think we should use `config.hideVersion`bool  rather than comparing strings. WDYT?

Reply:
🤔 Yeah, that makes sense. I'll apply that